ibis/__init__.py

```diff
@@ -29,7 +29,7 @@
 
 with suppress(ImportError):
     # pip install ibis-framework[impala]
-    import ibis.impala.api as impala  # noqa: F401
+    from ibis.backends import impala  # noqa: F401
```

Comment:
Tests are failing due to `ModuleNotFoundError: No module named 'ibis.impala'` - but i thought the above would have enabled that and the approach looked the same as OmniSci. am i missing something?

Reply:
There is something weird, and it's that the bigquery backend uses many things from the impala one (doesn't make sense, but there must be a historical reason for it). If you see the error, this is the import that is failing:
```
ibis/bigquery/compiler.py:17: in <module>
    from ibis.impala import compiler as impala_compiler
```
You'll have to update it to be `from ibis.backends.impala...`.